src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -473,6 +473,20 @@ $(document).ready(() => {
         showNewInstructorForm();
     });
 
+    $("[name='instructorisdisplayed']").change((e) => {
+        if ($(e.target).prop('checked')) {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('value', 'Instructor');
```

Comment:
Changing the `value` attribute does not seem to do anything (at least on Firefox), as the attribute `value` is always set to "Instructor" regardless of what the user enters as input.